src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Color color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);

```

Comment:
All magic numbers in your code should be constants. Please define '100' as a constant with a clear name indicating its purpose.